include/infinispan/hotrod/ConfigurationBuilder.h

```diff
@@ -21,10 +21,22 @@ using namespace infinispan::hotrod::event;
 namespace infinispan {
 namespace hotrod {
 
+/**
+ * ClusterConfigurationBuilder is used to configure the clusters that are part of the system.
+ * Is a builder of ClusterConfiguration objects.
+ */
 class ClusterConfigurationBuilder
 {
 public:
 	ClusterConfigurationBuilder(std::vector<ServerConfigurationBuilder>& servers, ConfigurationBuilder &parent) : servers(servers), m_parent(parent) {}
+	/**
+	 * Add a server node to this cluster
+	 *
+	 * \param host hostname
+	 * \param port tcp port number
+	 *
+	 * \return this object for further configuration
```

Comment:
You used "return this object for fluent configuration" below, and I think that is a better message